packages/twenty-front/src/modules/ui/layout/banner/components/BannerButton.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+
+const StyledBannerButton = styled.div`
```

Comment:
Not clickable and text selectable.

Should we re-use a button we already have in our UI ?